lib/hls/hls_parser.js

```diff
@@ -450,33 +482,34 @@ shaka.hls.HlsParser = class {
         }
       }
     }
-
-    this.streamsSynced_ = true;
   }
 
   /**
    * Synchronize streams by the EXT-X-PROGRAM-DATE-TIME tags attached to their
    * segments.  Also normalizes segment times so that the earliest segment in
-   * any stream is at time 0.
+   * the stream is at time 0.
```

Comment:
Perhaps I'm nit-picking, but I think the original version of this comment was more accurate, and I think it's a meaningful difference.  We don't make all streams start at 0.  We make the earliest segment in _any_ stream (minimum across streams) 0, but keep alignment.  Some streams may have their earliest segments start at a time > 0.

Reply:
Done.